### PR TITLE
State PHP requirement

### PR DESCRIPTION
**Pull request by szepeviktor**
## Description

Helps people using [Composer](https://github.com/szepeviktor/composer-managed-wordpress) to install WordPress.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "description": "GiveWP - a WordPress Donation Plugin and Fundraising Platform",
     "homepage": "https://givewp.com/",
     "require": {
+        "php": "^7.2 || ~8.0.0",
```
Comment (jonwaldstein) [inline composer.json:8]: We do already have a composer `platform` config set to php 7.2

Comment (szepeviktor) [reply composer.json:8]: Okay. What should I do now?

Comment (jonwaldstein) [reply composer.json:8]: If you agree this addition is not necessary since we already have the platform config, we can just close the PR

Comment (szepeviktor) [reply composer.json:8]: All right!

**Pull request closed without merging**